### ISSUE TITLE
WebSockets support

### ISSUE THREAD

**Issue by gimlet2** [gimlet2/kottpd#6]

**Comment by tipsy** [comment]
@gimlet2 are you still working on this project?

**Comment by gimlet2** [comment]
@tipsy Yes, now I'm working on support of async sockets. and some experiments with kotlin-native. But websockets on roadmap somewhere. Wanna help? =)

**Comment by tipsy** [comment]
> websockets on roadmap somewhere. Wanna help? =)

Happy to hear it. Sorry, I can't help, I need to prioritize working on my own framework ([javalin](https://javalin.io/)), but I really like your approach. My first thought was also that with this little code and no dependencies, you could go for kotlin-native.

I was particularly interested in this feature since I'm considering adding websocket support myself, and I'm curious to see how you'll solve it. I've put your project on watch :)
